.github/workflows/rcli-tap.yml

```
@@ -0,0 +1,78 @@
+            echo "Grant a token with push access to RunanywhereAI/homebrew-tap, then re-run"
+            echo "this workflow, or update by hand: ./rcli/scripts/update-tap.sh ${VERSION}"
+            exit 0
+          fi
+          export RCLI_TAP_REPO="https://x-access-token:${TAP_TOKEN}@github.com/RunanywhereAI/homebrew-tap.git"
+          ./rcli/scripts/update-tap.sh "${VERSION}"
+
+      # Proves the formula the tap now serves actually installs and runs, on the
+      # same OS a user would install from. `brew test` runs the formula's own
+      # test block, which checks `rcli version` and `rcli backends`.
+      - name: Verify brew install from the tap
+        if: ${{ secrets.RCLI_TAP_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)